app/src/org/commcare/android/adapters/MenuAdapter.java

```diff
@@ -87,9 +85,7 @@ public MenuAdapter(Context context, CommCarePlatform platform, String menuID){
                         try {
```

Comment:
Yeah, they were being caught but not displayed in any remotely helpful fashion. Fixed, should be good to go now.